Sming/SmingCore/SimpleTimer.h

```diff
@@ -0,0 +1,101 @@
+/****
+ * Sming Framework Project - Open Source framework for high efficiency native ESP8266 development.
+ * Created 2015 by Skurydin Alexey
+ * http://github.com/anakod/Sming
+ * All files of the Sming Core are provided under the LGPL v3 license.
+ *
+ *
+ * @author: 13 August 2018 - mikee47 <mike@sillyhouse.net>
+ *
+ * This class wraps the OS timer functions.
+ *
+ * For many timing operations the basic OS Timer capabilities are sufficient.
+ * The additional RAM required by the main Timer class soon adds up, so all
+ * short timer requirements are handled by an OSTimer. Still require Timer for
+ * longer periods and the additional callback flexibility.
+ *
+*/
+
+/** @defgroup   timer OSTimer functions
+ *  @brief      Provides timer functions
+*/
+
+#ifndef SMINGCORE_SIMPLETIMER_H_
```

Comment:
Can you change `SMINGCORE_` to `_SMING_CORE_`?

Reply:
Have also fixed guard for Timer.h